MAVProxy/modules/mavproxy_serial.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, mpstate):
         self.serial_settings = mp_settings.MPSettings(
             [ ('port', int, 0),
               ('baudrate', int, 57600),
-              ('timeout', int, 500)
```

Comment:
Shouldn't this fix be in ArduPilot itself?  That would usually be by flagging to the scheduler that we're going to be off doing other things, and is important because the hardware watchdog probably won't be being patted either without a fix in AP.